### PR TITLE
Tribute page

### PR DESCRIPTION
**Pull request by RiyaGupta89**
# Resolves issue #146 

Added tribute page project in the repository which is tribute page made for covid warriors. Added a readme.md. I cant find index.md in my local computer even after running git pull. Please help. I think this is some problem related to branch, if its onto some branch it only pulls code related to that branch or idk.

tribute

### CODE REVIEW

File: TRIBUTE PAGE/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+node app.js
+```
+- Have a look at the webpage.
+
+
+<h3> ScreenShots </h3>  
+<img src = "public/images/tribute.png" alt="Tribute-webpage-demo">
+<br>
+<img src = "public/images/tribute1.png" alt="Tribute-webpage-demo">
+
```
Comment (harshita2216) [inline TRIBUTE PAGE/README.md:34]: Images aren't being displayed.

**Review by harshita2216 — COMMENTED** [review]
Update index.md as well please. Make sure you have an updated repo, maybe then index.md would be visible. 
Also add a new line at the end of each page as asked by Mentor Shloka. 
Thanks.

File: JAVASCRIPT BUTTON APP/app.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+app.use(express.static("public"));
+
+app.get("/", function(req, res){
+    res.sendFile(__dirname + "/index.html");
+});
+
+app.listen(3000, function(req, res) {
+    console.log("Server started running on port 3000.");
+})
```
Comment (chicken-biryani) [inline JAVASCRIPT BUTTON APP/app.js:13]: New line at the eof

File: JAVASCRIPT BUTTON APP/index.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            <button class="btn btn-green">Push!</button>
+        </div>
+
+    </div>
+
+<script src="js/script.js"></script>
+
+</body>
+
+</html>
```
Comment (chicken-biryani) [inline JAVASCRIPT BUTTON APP/index.html:35]: New line at the eof

File: JAVASCRIPT BUTTON APP/public/css/style.css [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+.btn-green:hover{
+    background-color: rgb(8, 94, 8);
+}
+.btn-yellow{
+    background-color: rgb(240, 187, 41);
+    color: #000;
+}
+.btn-yellow:hover{
+    background-color: rgb(226, 168, 46);
+}
```
Comment (chicken-biryani) [inline JAVASCRIPT BUTTON APP/public/css/style.css:52]: new line at the eof

File: JAVASCRIPT BUTTON APP/public/js/script.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+function buttonRandom() {
+let choices = ["btn-red", "btn-green", "btn-yellow", "btn-blue"];
+for (let i = 0; i< all_buttons.length; i++) {
+let random = Math.floor(Math.random() * 4);
+all_buttons[i].classList.remove(all_buttons[i].classList[1]);
+all_buttons[i].classList.add(choices[random]);
+}
+}
```
Comment (chicken-biryani) [inline JAVASCRIPT BUTTON APP/public/js/script.js:51]: New line at the eof

**Review by chicken-biryani — COMMENTED** [review]
PLease add a new line at the end of every file

**Comment by RiyaGupta89** [comment]
Changes made in tribute page.

**Review by harshita2216 — COMMENTED** [review]
Add your project to the index.md file please!

**Comment by RiyaGupta89** [comment]
@harshita2216 Added. Please review.

**Review by harshita2216 — APPROVED** [review]

**Pull request merged**